config/checkstyle/checkstyle_reviewdog.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0"?>
```

Comment:
Do we really need to duplicate the checkstyle config? Makes it hard in the future to keep them synced

Reply:
Checks are also possible aka the default, but only visible if you look at the files changed tab. 
I personally think this comment variant is useful

Unfortunately we currently need this duplication because otherwise the suppression config file cannot be found due to the parameter expansion or the gradle check cannot be run.
There's also no way to inline the suppression

Reply:
It just adds so much noise, if you commit half-finished PRs that still need cleanup etc. You then get github notifications because a bot commented. Why not educate people to look at all checks?

Currently, the checkstyle issues do not show up in the changed files tab.

So with the current config, reviewdog will report about things that we actually decided to suppress?

Reply:
I changed back to github checks feature.

Again, for the copy. Reviewdog needs a copy of the checkstyle.xml because it contains the relative path to the supression.xml file. Otherwise it cannot parse the ${configloc} variable.
Gradle and the IDEs need that  variable to automatically set the correct path

Reply:
Thanks!

Can you then please open an issue at reviewdog so that this gets fixed in the future. Thanks!

Reply:
It's more a checkstlye issue (that won't be fixed). However, I opened an issue at the action repo.

Reply:
The issue is https://github.com/nikitasavinov/checkstyle-action/issues/16. It should have been solved with https://github.com/nikitasavinov/checkstyle-action/pull/19. Should we try it?

Reply:
Try it...